swift/storage.py

```diff
@@ -357,11 +355,8 @@ def _path(self, name):
         # Are we building a temporary url?
         if self.use_temp_urls:
             expires = int(time() + int(self.temp_url_duration))
-            method = 'GET'
             path = urlparse.urlsplit(url).path
```

Comment:
Should be :

    path = urlparse.unquote(urlparse.urlsplit(url).path)

..because `name` was quoted above (line 355).